implementation/jobberknoll/infra/deno.json

```diff
@@ -1,5 +1,4 @@
 {
   "name": "@jobberknoll/infra",
-  "version": "0.1.0",
```

Comment:
Consolidating separate versions for subpackages in the workspace to a single workspace version.